src/vm/interpreter.cpp

```diff
@@ -11817,7 +11817,7 @@ void Interpreter::PrintValue(InterpreterType it, BYTE* valAddr)
                 {
                     fprintf(GetLogFile(), " ");
                 }
-                fprintf(GetLogFile(), "0x%p", valAddr[i]);
+                fprintf(GetLogFile(), "0x%b", valAddr[i]);
```

Comment:
%p leads to error being treated as warning.

'fprintf'  : '%p' in format string conflicts with argument 1 of type 'int'